README.md

```diff
@@ -30,13 +30,32 @@ $ operator-courier push $BUNDLE_DIR $EXAMPLE_NAMESPACE $EXAMPLE_REPOSITORY $EXAM
 
 Once that is created, you should be able to view your bundle on quay.io's Application page for your particular namespace, repo, and release version (https://quay.io/application/$EXAMPLE_NAMESPACE/$EXAMPLE_REPOSITORY?tab=$EXAMPLE_RELEASE)
 
-For more info, run help on any of the subcommands
+For more info, run help on the main program or any of the subcommands
 
 ```bash
 $ operator-courier -h
-$ operator-courier push -h
+$ operator-courier SUBCOMMAND -h
 ```
 
+### Source Directory Structure
+
+Currently operator-courier supports source directory in both **flat** and **nested** structure.
+
+- **flat directory** includes CRD, CSV and package files all in one single directory
```

Comment:
IMO we shouldn't advertise the flat dir structure. We have it for legacy purposes but all of our docs should be pointing to the nested operator-registry format.

Reply:
@kevinrizza But in the current README in the [CLI](https://github.com/operator-framework/operator-courier/blob/master/README.md#command-line-interface) and [Library](https://github.com/operator-framework/operator-courier/blob/master/README.md#library) section, they only assume the source directory to contain our old flat directory format. Should I also update those and replace them with the new nested format?

Reply:
@jeremylinlin oh yeah that's a good point. We should update those places in the readme as well.